src/org/jitsi/util/RTPUtils.java

```diff
@@ -44,6 +44,7 @@ else if (diff > 1<<15)
      * (modulo 2^16).
      * @return result of the subtraction of one RTP sequence number from another
      * (modulo 2^16).
+     * FIXME(brian): why do we have this and sequenceNumberDiff?
```

Comment:
so are there still valid cases for both to exist?  should i rename `subtractNumber` to something else?